src/tsickle.ts

```diff
@@ -744,12 +740,11 @@ class Annotator extends ClosureRewriter {
     if (!classDecl.name) return;
     let className = getIdentifierText(classDecl.name);
 
-    this.emit('\n\n  static _tsickle_typeAnnotationsHelper() {\n');
```

Comment:
I think we discussed it at the time. I vaguely think we considered it, but then this worked and seemed closer to what ought to happen.